Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -215,6 +284,25 @@ public KeyCommand()
             }
         }
 
+        [Command(Description = "Derive Public key and Address from Private key.")]
+        public void Derive(
+            [Argument(
+                "PRIVATE-KEY",
+                Description = "A raw private key to import."
+            )]
+            string key,
+            PassphraseParameters passphrase
+        )
+        {
+            PrivateKey privateKey = ValidateRawHex(key);
+            string passphraseValue = passphrase.Take("Passphrase: ");
+            string addr = privateKey.ToAddress().ToString();
+            string pub = ByteUtil.Hex(privateKey.PublicKey.Format(compress: true));
+            ProtectedPrivateKey ppk = ProtectedPrivateKey.Protect(
+                privateKey, passphraseValue);
+            Utils.PrintTable(("Public Key", "Address"), new[] { (pub, addr) });
+        }
```

Comment:
If I understood well, the `passphrase` variable and the `ppk` variable seem not useless in the `derive` command. Because it doesn't print the protected private key or store it. What do you think about it?

Reply:
Yep, totally unused. removed unused ppk and passphrase.